Sources/muterCore/TestReporting/TestSuiteOutcome.swift

```diff
@@ -21,9 +21,9 @@ public enum TestSuiteOutcome: String, Codable {
 }
 
 extension TestSuiteOutcome {
-    public static func from(testLog: String) -> TestSuiteOutcome {
+    public static func from(testLog: String, terminationStatus: Int32) -> TestSuiteOutcome {
 
-        if logContainsRuntimeError(testLog) {
+        if !terminationStatusIsSuccess(terminationStatus) {
```

Comment:
I think regression tests are still passing, right? Because I think `xcodebuild` returns `65` when tests fails.
I've find this [unofficial list of exit codes](https://unix.stackexchange.com/questions/110348/how-do-i-get-the-list-of-exit-codes-and-or-return-codes-and-meaning-for-a-comm)... if can also improve it later